app/cdap/components/common/DrawerWidget/index.tsx

```diff
@@ -107,7 +108,6 @@ export default function({
   onBackIconClick,
   anchor = 'right',
   dataTestId,
-  showHeaderSeparator = true,
```

Comment:
Why is this removed? As Drawer widget is  common for V2 & V3, added this check to show the Header Separator.  Please check once , otherwise import Recipe components may fail to compile.

Reply:
It was a mistake, as per design for recipe details panel, didn't see a separator, that's why removed it, Added it again

Reply:
As per v3 design we don't have underline img for Heading Title in entire application so removed showHeaderSeparator prop  as we don't require it anymore